src/libraries/Native/Unix/System.Security.Cryptography.Native/openssl.c

```diff
@@ -1319,6 +1319,13 @@ static int32_t EnsureOpenSsl11Initialized()
 
 #endif
 
+int32_t CryptoNative_OpenSslAvailable()
+{
+    // OpenLibrary will attempt to open libssl. DlOpen will handle
+    // the case of it already being open and dlclose the duplicate
+    return OpenLibrary();
```

Comment:
```suggestion
#ifdef FEATURE_DISTRO_AGNOSTIC_SSL
    // OpenLibrary will attempt to open libssl. DlOpen will handle
    // the case of it already being open and dlclose the duplicate
    return OpenLibrary();
#else
    return 1;
#endif
```

Reply:
I was having a similar thought, but got distracted while looking up how opensslshim.c gets built in portable mode.